#42 Implement Dashboard blueprint / Implement Dashboard content #43

ArinzeJeffrey-droid opened the pull request:
Fixes #42 
Fixes #43

toystars commented:
@ArinzeJeffrey-droid could you please provide screenshots? Also, when this #38 gets merged, some changes will be required.

ArinzeJeffrey-droid commented:
> @ArinzeJeffrey-droid could you please provide screenshots? Also, when this #38 gets merged, some changes will be required.

Okay, I'll provide the screenshot. Also, I followed the file structure pattern based of #38 pr.

ArinzeJeffrey-droid commented:
![mentorpaired](https://user-images.githubusercontent.com/61787049/100081058-6e917600-2dfb-11eb-847c-d7d85ed48a8d.PNG)
This is how it looks. I also implemented basic routing to different components ( Dashboard, Profile, Message ), and i added the active state funtionality. @toystars

toystars commented:
@ArinzeJeffrey-droid there are a few difference between the design and the screenshot provided.

- Only selected side navigation item should have a blue stripe on the right. In the screenshot you provided, all the items have a right border.
- The blue stripe should be on the border of the sidebar without any margin.
- The `Mentopaired` logo is wrong. You should be able to export the logo as an svg from the Figma design file.

toystars commented on src/modules/dashboard/components/navbar/Navbar.jsx:
```diff
@@ -0,0 +1,41 @@
+      </ul>
+      <div className="profile-nav">
+        <div className="bell">
+          <BellLogo/>
```
Why are these svgs rendered as React components and not used as images?

ArinzeJeffrey-droid replied:
I rendered it as a component just in case we implement the notification alert (the blue dot), it'll be easier to set if it's active or not.

toystars replied:
Got it

ArinzeJeffrey-droid commented:
> @ArinzeJeffrey-droid there are a few difference between the design and the screenshot provided.
> 
> * Only selected side navigation item should have a blue stripe on the right. In the screenshot you provided, all the items have a right border.
> * The blue stripe should be on the border of the sidebar without any margin.
> * The `Mentopaired` logo is wrong. You should be able to export the logo as an svg from the Figma design file.

I'll get rid of the right border, but the reason why the first one is blue is because it's selected.
The "Mentorpaired" logo is an svg.

ArinzeJeffrey-droid commented:
![Screenshot (1)](https://user-images.githubusercontent.com/61787049/100332927-d922ed00-2f86-11eb-8147-595e8d5375f6.png)

Made the adjustments as requested.

toystars commented:
> ![Screenshot (1)](https://user-images.githubusercontent.com/61787049/100332927-d922ed00-2f86-11eb-8147-595e8d5375f6.png)
> 
> Made the adjustments as requested.

@ArinzeJeffrey-droid the spacing between the side navigation items doesn't align with what is in the design. They need to be closer than they are right now. I'll fully review the code afterwards.

devraulu commented:
![image](https://user-images.githubusercontent.com/67712782/100552339-07d9e580-325d-11eb-968c-813cfa267e23.png)

This notifications bar should not show up in the profile section.

ArinzeJeffrey-droid commented:
> ![image](https://user-images.githubusercontent.com/67712782/100552339-07d9e580-325d-11eb-968c-813cfa267e23.png)
> 
> This notifications bar should not show up in the profile section.

Okay I'll get it fixed. Thanks.

devraulu commented:
![image](https://user-images.githubusercontent.com/67712782/100656958-ec85dd80-3323-11eb-9ef0-3ac0d194f639.png)
Since we are working with the PC version first, I think we should make the page responsive starting from 1024px and up, as you can see the sidebar is kind of big in that viewport width. Maybe that's something for discussion. @toystars @ayoola-solomon

ArinzeJeffrey-droid commented:
> ![image](https://user-images.githubusercontent.com/67712782/100656958-ec85dd80-3323-11eb-9ef0-3ac0d194f639.png)
> Since we are working with the PC version first, I think we should make the page responsive starting from 1024px and up, as you can see the sidebar is kind of big in that viewport width. Maybe that's something for discussion. @toystars @ayoola-solomon

It does looks really big, I'll also make the adjustments.

ArinzeJeffrey-droid commented:
Made all the required fixes.

ArinzeJeffrey-droid commented:
![Screenshot (4)](https://user-images.githubusercontent.com/61787049/101464092-11b1b780-38f3-11eb-9c52-e4e0a4194e70.png)
Fixed navigation bar on profile page.
![Screenshot (5)](https://user-images.githubusercontent.com/61787049/101464196-30b04980-38f3-11eb-99f4-5641b1c48007.png)
And also changed the em/rem values to pixels.
